megamek/src/megamek/common/BattleArmorHandles.java

```diff
@@ -14,6 +14,7 @@
 
 package megamek.common;
 
+import java.util.ArrayList;
```

Comment:
Seems unnecessary.

Reply:
Thanks. Remnant of a piece of code I had added to the Transporter interface that I wound up not needing.